docker/fullnode/docker-compose.yaml

```diff
@@ -2,15 +2,16 @@
 version: "3.9"
 services:
   fullnode:
-    image: mysten/sui-node:stable
+    image: mysten/sui-node:mainnet-v1.19.1
```

Comment:
this will get stale quick - maybe we can convince @ebmifa to add back the `mainnet` and `testnet` docker image tags

Reply:
Totally agree. I will add a comment here, and yes it will be nice if we provide up-to-date `mainnet` and `testnet` tags.

Reply:
Can't we get that tag dynamically? 
The community has asked a few times to use `mainnet-v1.19.1` like tags for docker hub, so that it matches how the releases are referenced in GitHub. Also, what does `stable` even mean in this context?

Reply:
https://github.com/MystenLabs/sui/pull/16663 will start using `devnet`, `testnet` and `mainnet` tags